openassessment/workflow/api.py

```diff
@@ -342,11 +342,9 @@ def _get_workflow_model(submission_uuid):
         raise AssessmentWorkflowRequestError("submission_uuid must be a string type")
 
     try:
-        workflow = AssessmentWorkflow.objects.get(submission_uuid=submission_uuid)
-    except AssessmentWorkflow.DoesNotExist:
-        raise AssessmentWorkflowNotFoundError(
-            u"No assessment workflow matching submission_uuid {}".format(submission_uuid)
-        )
+        workflow = AssessmentWorkflow.get_by_submission_uuid(submission_uuid)
```

Comment:
Note to reviewers: I was just trying to reuse the get_by_submission_uuid method, while retaining the original error handling of this method. I'm not sure if this was worth it though.... Happy to back out this change.